examples/transloadit/index.html

```diff
@@ -27,103 +27,107 @@
 
       <p>
-        The form API allows you to easily send files through Transloadit's encoding backend. When the user submits the form, any files are uploaded to Transloadit. The form data is then sent to your own backend, with additional data about the Transloadit Assemblies that were started.
-
+        The form API allows you to easily send files through Transloadit’s encoding backend. When the user submits the form, any files are uploaded to Transloadit. The form data is then sent to your own backend, with additional data about the Transloadit Assemblies that were started.
+      </p>
       <form id="test-form" method="post" action="http://localhost:9967/test">
-        <p><strong>leave a message</strong>
+        <p><strong>leave a message</strong></p>
         <p>
           <label>name:
             <input type="text" name="name">
           </label>
+        </p>
         <p>
           <label>message: <br>
             <textarea name="message"></textarea>
           </label>
-
+        </p>
         <p>
           <label>
             attachments:
-            <input type="file" name="files" multiple>
+            <!-- <input type="file" name="files" multiple> -->
+            <strong id="uppy-form-selected-files"></strong>
+            <button type="button" id="uppy-select-files">Select Files</button>
           </label>
           <div class="progress"></div>
-
+        </p>
         <p>
           <button type="submit">
             Upload
           </button>
-
+        </p>
           <span class="error"></span>
       </form>
 
       <hr>
-      <h2>robodog.form() with dashboard</h2>
+      <h2>Form with inline Dashboard</h2>
 
       <p>
-        You can also use the Dashboard UI inside a plain old HTML form by specifying a <code>dashboard: '.target-css-selector'</code> option.
-
+        You can also use the Dashboard UI inside a plain old HTML form.
+      </p>
       <form id="dashboard-form" method="post" action="http://localhost:9967/test">
-        <p><strong>leave a message</strong>
+        <p><strong>leave a message</strong></p>
         <p>
           <label>name:
             <input type="text" name="name">
           </label>
+        </p>
         <p>
           <label>message: <br>
             <textarea name="message"></textarea>
           </label>
-
+        </p>
         <p>
           <label>
             attachments:
             <span class="dashboard"></span>
           </label>
-
+        </p>
         <p>
           <button type="submit">
             Upload
           </button>
-
+        </p>
           <span class="error"></span>
       </form>
 
       <hr>
-      <h2>robodog.dashboard()</h2>
-
+      <h2>Inline Dashboard</h2>
       <p>
         The <code>robodog.dashboard</code> API allows you to embed a Dashboard at any location. Users can continuously upload files through this UI, so please make sure this fits your use case!
-
-      <p id="dashboard">
+      </p>
+      <p id="dashboard"></p>
```

Comment:
That's definitely not a paragraph.
```suggestion
      <div id="dashboard"></div>
```